### PR TITLE
A4TECH.Bloody7 version V2022.0805

### PR DESCRIPTION
**Pull request by Debtless**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/89606)

### CODE REVIEW

File: manifests/a/A4TECH/Bloody7/V2022.0805/A4TECH.Bloody7.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+PackageIdentifier: A4TECH.Bloody7
+PackageVersion: V2022.0805
+Installers:
+- Architecture: x86
+  InstallerType: exe
+  InstallerUrl: https://www.a4tech.com.tw/download/BloodyMouse/Bloody7_V2022.0805_MUI.exe
+  InstallerSha256: 7D02D9F654A9B60FAE5E7C22D493D8EDA4711AA5E4CC4296A6583AD544334550
+  InstallerSwitches:
+    Silent: /qn
```
Comment (Trenly) [inline manifests/a/A4TECH/Bloody7/V2022.0805/A4TECH.Bloody7.installer.yaml:12]: Please also include the `SilentWithProgress` switch

image

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/a/A4TECH/Bloody7/V2022.0805/A4TECH.Bloody7.installer.yaml (hunk 1 above)
Comment (Debtless) [reply manifests/a/A4TECH/Bloody7/V2022.0805/A4TECH.Bloody7.installer.yaml:12]: > Please also include the `SilentWithProgress` switch
> 
What flag should I add to include the "SilentWithProgress" switch?

Comment (stephengillie) [reply manifests/a/A4TECH/Bloody7/V2022.0805/A4TECH.Bloody7.installer.yaml:12]: The `SilentWithProgress` switch might be the same as the `Silent` switch.

I believe the issue Trenly is seeing is actually caused by the installer being launched by a self-extracting Winrar archive. The `Silent` switch /S hides the unpacking operation, but the installer is still in attended mode. After copying the installer files to another location and querying it, I couldn't find any switches to enable unattended mode, nor even a dialog of available switches. Unfortunately, packages must have a fully unattended install mode to be available through Winget.

**Comment by stephengillie** [comment]
Unfortunately, installers must have a fully unattended mode to be included in the Winget manifest.

**Pull request closed without merging**